symengine/ruby/spec/basic_spec.rb

```diff
@@ -16,56 +16,5 @@
                 end
             end
         end
-
-        describe 'binary operations' do
-            before :each do
-                @a = SymEngine::Basic.new
-                @b = SymEngine::Basic.new
-            end
-            describe '#+' do
-                context 'with another Basic object as argument' do
-                    xit 'returns a Basic object that is result of
-                        self added to the argument' do
```

Comment:
What I meant was since `Symbol` is a `Basic`, think of it as a `Basic`. Instead of `@a = SymEngine::Basic.new` you could use  `a = SymEngine::Symbol.new("a")`